sandbox/sandbox_linux.go

```diff
@@ -0,0 +1,161 @@
+//go:build linux
+
+package sandbox
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/projectdiscovery/gozero/command"
+	stringsutil "github.com/projectdiscovery/utils/strings"
+)
+
+type Configuration struct {
+	Rules []Rule
+}
+
+type Filter string
+
+const (
+	PrivateTmp              Filter = "PrivateTmp"
+	PrivateNetwork          Filter = "PrivateNetwork"
+	SELinuxContext          Filter = "SELinuxContext"
+	NoNewPrivileges         Filter = "NoNewPrivileges"
+	ProtectSystem           Filter = "ProtectSystem"
+	ProtectHome             Filter = "ProtectHome"
+	ProtectDevices          Filter = "ProtectDevices"
+	CapabilityBoundingSet   Filter = "CapabilityBoundingSet"
+	ReadWriteDirectories    Filter = "ReadWriteDirectories"
+	ReadOnlyDirectories     Filter = "ReadOnlyDirectories"
+	InaccessibleDirectories Filter = "InaccessibleDirectories"
+	ProtectKernelTunables   Filter = "InaccessibleDirectories"
+	ProtectKernelModules    Filter = "ProtectKernelModules"
+	ProtectControlGroups    Filter = "ProtectControlGroups"
+	RestrictNamespaces      Filter = "RestrictNamespaces"
+	MemoryDenyWriteExecute  Filter = "MemoryDenyWriteExecute"
+	RestrictRealtime        Filter = "RestrictRealtime"
+	PrivateMounts           Filter = "PrivateMounts"
+	DynamicUser             Filter = "DynamicUser"
+	SystemCallFilter        Filter = "SystemCallFilter"
+)
```

Comment:
@Mzack9999 , in future are we going to use these variables outside of `gozero`

- if not then I think we can move them into `internal` package to avoid any unforeseen situations
- If yes then I think it would be better if we place all these constants/variables in `constants.go` to avoid using os specific build files in gozero derivatives when using these options
-

Reply:
What do you think about having a runtime map populated with with values specific for the OS, we could initialize them in the `init()` function similarly to https://github.com/projectdiscovery/nuclei/pull/3421

Reply:
yup I think it makes more sense to do that way

Reply:
After doing some tests, I'm not sure that moving them out of conditional build would bring any benefit as they are specific to the OS platform, and, in my opinion, they should remain scoped as that, failing to compile if used under the wrong system. We should later implement a generic parser/marshaler that converts them from a generic rule into the specific OS configuration dialect file (windows/osx) or cli invocation (linux). If you agree, I'd suggest approaching this after the docker/nixos integration.